infra-templates/k8s/57/rancher-compose.yml

```diff
@@ -0,0 +1,348 @@
+.catalog:
+  name: Kubernetes
+  version: v1.12.5-rancher1-2
```

Comment:
it looks like in the past we've done something like we've appended a `-1` when we have the same image but changes in the template. so this should be `v1.12.5-rancher1-1-1`